gdpr/anonymizers/model_anonymizers.py

```diff
@@ -292,36 +299,80 @@ def anonymize_qs(self, qs: QuerySet) -> None:
     def deanonymize_qs(self, qs: QuerySet) -> None:
         raise NotImplementedError()
 
+    def get_anonymizer_or_none(self, name):
+        return self.anonymizers.get(name, None)
+
+    def update_related_anonymizer_fields(self, field_name:str, obj: Model, related_fields:Fields,
+                                         legal_reason: Optional[LegalReason] = None,
+                                         purpose: Optional["AbstractPurpose"] = None,
+                                         anonymization: bool = True):
+        anonymizer = self.get_anonymizer_or_none(field_name)
+        related_attribute = getattr(obj, field_name, None)
+
+        if hasattr(obj.__class__, field_name) and related_attribute is None:
+            return
+        elif anonymizer and anonymizer.is_relation:
+            objs = self.get_anonymizer_or_none(field_name).get_related_objects(obj)
+            for related_obj in objs:
+                related_fields.anonymizer.update_obj(
+                    related_obj, legal_reason, purpose, related_fields,
+                    base_encryption_key=self._get_encryption_key(obj, field_name),
+                    anonymization=anonymization
+                )
+        elif isinstance(related_attribute, self.model):
+            related_fields.anonymizer.update_obj(
+                related_obj, legal_reason, purpose, related_fields,
```

Comment:
`related_obj` is undefined here, probably should be `related_attribute`

Reply:
@matllubos tests missing for this one?

Reply:
It seems that yes, I will add it